spring-cloud-gateway-server/src/test/java/org/springframework/cloud/gateway/filter/factory/RetryGatewayFilterFactoryIntegrationTests.java

```diff
@@ -109,6 +109,18 @@ public void retryWithBackoff() {
 		// @formatter:on
 	}
 
+	@Test
+	public void retryWithBackoffJitterTimeout() {
+		// @formatter:off
+		testClient.get()
```

Comment:
Is this really testing the timeout functionality?  It looks like `/retry` just throws an exception.  The jitter would be hard to test and verify but we should be able to test the timeout.

Reply:
ok. i have added an other unit-test to test the timeout functionality